src/lib/components/ListingPageLayout.svelte

```
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import Info from '@lucide/svelte/icons/info';
+	import Label from '$lib/components/ui/label/label.svelte';
+
+	type ListingPageLayoutProps = {
+		title: string;
+		subtitle: string;
+		showFilters?: boolean;
+		showEmptyState?: boolean;
+		showInfoIcon?: boolean;
+		headerActions?: () => any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)